lab-michael/gulpfile.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+
+
+gulp.task('welcome', function() {
+  console.log('Welcome to the HTTP SERVER TEST');
+});
+
+gulp.task('test', function() {
+  gulp.src('./test/test-*.js', {read: false})
+  .pipe(mocha({reporter: 'nyan'}));
+});
+
+gulp.task('eslint', function() {
+  return gulp.src(['**/*.js', '!node_modules/**'])
+  .pipe(eslint({ fix: true}))
```

Comment:
Be cautious of letting eslint fix all  your errors by default. It is only just a box of electrical 1s and 0s. :-)